x/cert/keeper/keeper.go

```diff
@@ -59,18 +59,13 @@ func (k keeper) CreateCertificate(ctx sdk.Context, owner sdk.Address, crt []byte
 		return types.ErrCertificateExists
 	}
 
-	iter := sdk.KVStorePrefixIterator(store, certificatePrefix(owner))
-	defer func() {
-		_ = iter.Close()
-	}()
```

Comment:
The `iter` was declared but wasn't used. Seemed unnecessary, so removed.

Reply:
🙏